service/backend/pchain/account.go

```diff
@@ -291,7 +290,7 @@ func (b *Backend) fetchBalance(ctx context.Context, addrString string, fetchImpo
 // Copy of the platformvm service's GetBalance implementation.
 // This is needed as multisig UTXOs are cleaned in parseUTXOs and its output must be used for the calculations. Ref:
 // https://github.com/ava-labs/avalanchego/blob/0950acab667e0c16a55e9a9bb72bcbe25c3b88cf/vms/platformvm/service.go#L184
-func (b *Backend) getBalancesWithoutMultisig(utxos []avax.UTXO) (*AccountBalance, error) {
+func (*Backend) getBalancesWithoutMultisig(utxos []avax.UTXO) (*AccountBalance, error) {
```

Comment:
nit: in cases where we aren't adhering to an interface, I think we should make these functions rather than methods.